packages/historical-sync/src/index.ts

```
@@ -1,96 +1,11 @@
-import { fetchPriceSnapshot } from '@pokt.metrics/indexer/dist/fetchers';
-import { storePriceSnapshots } from '@pokt.metrics/indexer/dist/services';
-import { Chain } from '@pokt.metrics/indexer/dist/utils/chains';
-import { baseClient, ethereumClient, solanaClient } from '@pokt.metrics/indexer/dist/utils/helpers';
+import { runIndexer } from '@pokt.metrics/indexer/dist';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)